pkg/lwip/contrib/netdev/lwip_netdev.c

```diff
@@ -49,6 +49,9 @@
 #define ETHERNET_IFNAME1 'E'
 #define ETHERNET_IFNAME2 'T'
 
+#define SIXLOWPAN_IFNAME1 'S'
+#define SIXLOWPAN_IFNAME2 'L'
```

Comment:
Woud prefer the following naming.

```suggestion
#define WPAN_IFNAME1      'W'
#define WPAN_IFNAME2      'P'
```

6LoWPAN is the network adaption layer protocol (which also can be used e.g. for IPv6 over BLE), the link layer technology here is IEEE 802.15.4 (also known as WPAN).